azure-pipelines.yml

```diff
@@ -190,6 +190,20 @@ jobs:
       modifyEnvironment: true
     condition: and(succeeded(), eq(variables['vmImage'], 'windows-2019'))
 
+  - powershell: |
+      git clone https://github.com/Microsoft/vcpkg.git
+      if ($LASTEXITCODE -ne 0) { throw "git clone failed" }
+      cd vcpkg
+      .\bootstrap-vcpkg.bat
+      if ($LASTEXITCODE -ne 0) { throw "bootstrap-vcpkg failed" }
+      .\vcpkg integrate install
+      if ($LASTEXITCODE -ne 0) { throw "vcpkg integrate failed" }
+      .\vcpkg install opencv:x64_windows
```

Comment:
.\vcpkg install opencv:x64-windows

Reply:
- not _

Reply:
Hmm, I have OpenCV installed, but I find_package fails to find it.

Reply:
Fixed the first problem, but find_package not finding it is concerning.